gradle/javaLibsProject.gradle

```diff
@@ -50,17 +49,16 @@ dependencies {
     dist 'org.apache.geronimo.specs:geronimo-activation_1.1_spec:1.1'
     dist 'org.apache.geronimo.specs:geronimo-stax-api_1.0_spec:1.0.1'
     dist 'com.google.code.gson:gson:2.7'
-    dist 'com.google.guava:guava:19.0'
     dist 'com.github.jknack:handlebars:4.0.6'
     dist 'com.sun.mail:javax.mail:1.6.2'
     dist 'jaxen:jaxen:1.1.6'
     dist 'io.netty:netty-buffer:4.1.39.Final'
-    dist 'io.netty:netty-codec-http2:4.1.39.Final'
-    dist 'io.netty:netty-codec-http:4.1.39.Final'
+    dist 'io.netty:netty-codec-http2:4.1.59.Final'
```

Comment:
Do we still need netty dependencies in ballerina-lang?

Reply:
There are some places these are still referred like here and even in some tests 
https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-core/build.gradle#L31

Reply:
Anyway we need to clean the whole repo and distribution packing dependencies with another effort.
https://github.com/ballerina-platform/ballerina-lang/issues/30853